README.md

```diff
@@ -6,15 +6,29 @@ A generic, reusable [web component](https://developer.mozilla.org/en-US/docs/Web
 
 ### User flow
```

Comment:
I think above this, it could be useful to add in links to the Google Docs (Getting Started guide, dashboard guide, data analysis guide)